src/apps/recommender/Recommender.tsx

```diff
@@ -77,7 +77,7 @@ const Recommender: FC = () => {
   const stillLoading = fbsReservationsLoading || fbsLoansLoading;
 
   return (
-    <div className="recommender">
+    <div className="recommender recommender--padding-desktop recommender--triple-desktop">
```

Comment:
Should double dash not be double underscores?

Reply:
I think this is correct, bem modifiers and not elements.